assets/js/params/Custom/Filters/Condition.js

```diff
@@ -64,7 +64,7 @@ export class Condition extends React.PureComponent {
         value={value}
         onChange={this.handleChange}
       >
-        <option value=''>{i18n._(t('js.params.Custom.Filters.Condition.selectCondition')`Select condition`)}</option>
+        <option value=''>{i18n._(/* i18n: As in 'select a condition relative to which we will filter'. Condition here is something of the form 'IF something IS like that'. */t('js.params.Custom.Filters.Condition.selectCondition')`Select condition`)}</option>
```

Comment:
A general query about this param: reading this comment, I realize the ordering of form elements -- `IF`, `[columns]`, `[behaves like]`, `[value]` -- may be ... English. Is that the case? Does the form design itself make sense in Greek?

Reply:
It makes complete sense in Greek